### DISCUSSION TITLE
Advent of Code 2017

### DISCUSSION THREAD

**Discussion by Sophmrs** [frontendbr/forum#836] [Dicas]
E aí, alguém participando do Advent of Code esse ano? Postem aí suas soluções.

Para quem não conhece, o [Advent of Code](https://adventofcode.com/) é uma série de puzzles de programação que são postados um por dia em dezembro até o natal, que acontece desde 2015.

Os problemas tem uma boa range de dificuldade, com um pouco pra todo mundo, e eles podem ser resolvidos em qualquer linguagem (ou até na mão), já que ele só pede o resultado para o seu input randomizado.

Vou estar postando os meus códigos deste ano ao longo do mês num [repo aqui no github.](https://github.com/Theomg/Advent-of-Code/tree/master/2017)

PS: Pra quem for fazer em javascript ou typescript o [Quokka](https://quokkajs.com/) é uma extensão bem legal pra usar editor como scratchpad.

**Comment by aajjbb** [comment]
to postando minhas solucoes nesse [repo](https://github.com/aajjbb/adventofcode-2017)

**Comment by TiagoDanin** [comment]
Crie um rank privado para nós!!!
Code: `211529-24efcbd7`
Só coloca o code aqui: http://adventofcode.com/2017/leaderboard/private

**Comment by felipefialho** [comment]
Não conhecia, valeu a dica @Theomg
